sandcastle/api.py

```diff
@@ -59,11 +60,30 @@
     SandcastleExecutionError,
     SandcastleTimeoutReached,
 )
-from sandcastle.utils import get_timestamp_now, clean_string
+from sandcastle.kube import PVC
+from sandcastle.utils import (
+    get_timestamp_now,
+    clean_string,
+    run_command,
+    purge_dir_content,
+)
 
 logger = logging.getLogger(__name__)
 
 
+class MappedDir:
+    """
+    Copy local directory to the pod using `oc cp`
+    """
+
+    def __init__(self, local_dir: str, path: str, with_interim_pvc: bool):
+        # path within the sandbox where the directory should be copied
+        self.path = Path(path)
+        # copy this local directory to sandbox (using `oc cp`)
+        self.local_dir = Path(local_dir)
+        self.with_pvc = with_interim_pvc
```

Comment:
Why not the same name?

Reply:
Well... I don't know to be honest. But I guess that you're right, default to True makes more sense, b/c otherwise people would need to copy things to /tmp, because that's the only writable place in a pod.